docs(ddx): add doc for disk mounting (#5764)
scaleway/docs-content: pages/dedibox-scaleway/reference-content/mount-fs-in-resuce-mode.mdx

---
title: Manually mounting the filesystem of your Dedibox in rescue mode
description: This page explains how to manually mount the filesystem of your Dedbox in rescue mode to acces your data.
dates:
  validation: 2025-11-05
  posted: 2025-11-05
tags: dedibox scaleway
---

This guide explains how to manually mount your server’s filesystem when running in [rescue mode](/dedibox/how-to/use-rescue-mode/).

> This procedure is commonly used to diagnose and repair issues on the root disk (e.g., filesystem corruption, kernel upgrade failure, lost credentials).

Reviewer: ```suggestion
This procedure is commonly used to diagnose and repair issues on the root disk (e.g., filesystem corruption, kernel upgrade failure, lost credentials, etc.).
```

     Always run repair tools on unmounted filesystems.
   </Message>

   ### ext4
   ```bash
   fsck -f /dev/sda2
   ```

   ### XFS
   ```bash
   xfs_repair /dev/sda2
   ```

   ### Btrfs
   ```bash
   btrfs check --repair /dev/sda2
   ```

5. Create a mount point:
   ```bash
   mkdir -p /mnt/root
   ```

6. Mount the root filesystem.
   Replace `/dev/sda2` with your actual root partition:
   ```bash
   mount /dev/sda2 /mnt/root
   ```
   Verify the mount:
   ```bash
   ls /mnt/root
   ``
   If you have a separate `/boot` (often `sda1`):
   ```bash
   mount /dev/sda1 /mnt/root/boot
   ```
   For UEFI systems with an EFI System Partition (ESP):
   ```bash
   mount /dev/sda1 /mnt/root/boot/efi
   ```

7. (Optional) Mount additional partitions.
   Common examples:
   ```bash
   mount /dev/sda3 /mnt/root/home
   mount /dev/sda4 /mnt/root/var
   ```

8. Bind system directories (required for chroot).
   ```bash
   mount -t proc none /mnt/root/proc
   mount -t sysfs none /mnt/root/sys
   mount --bind /dev /mnt/root/dev
   mount --bind /run /mnt/root/run
   ```

9 Enter the chroot environment to perform your desired rescue actions:
   ```bash
   chroot /mnt/root
   ```

> 9. Unmount and clean up once you are done.

Reviewer 2: ```suggestion
10. Unmount and clean up once you are done.
```

   Exit chroot first (if used):
   ```bash
   exit
   ```

   Then unmount everything recursively:
   ```bash
   umount -R /mnt/root
   ```

   Remove the temporary directory:
   ```bash
   rmdir /mnt/root
   ```

---

## Troubleshooting

### Mount fails with “wrong fs type”
Run the following command to identify the filesystem type of the partition:
```bash
blkid /dev/sdaX
```
Then specify the filesystem type explicitly:
```bash
mount -t ext4 /dev/sdaX /mnt/root
```

### Encrypted partitions (LUKS)

> If you partition is encrypted, you have to open it prior to mounting it:

Reviewer 2: ```suggestion
If your partition is encrypted, you have to open it before mounting it:
```
